domain/secret/state/state.go

```diff
@@ -1964,15 +2018,18 @@ WHERE  rev.secret_id = $secretInfo.secret_id`
 		return nil, 0, errors.Trace(domain.CoerceError(err))
 	}
 	if len(dbSecretConsumers) == 0 {
-		return nil, latestRevision, fmt.Errorf("secret consumer for %q and unit %q%w", uri.ID, unitName, secreterrors.SecretConsumerNotFound)
+		return nil, latestRevision, fmt.Errorf(
+			"secret consumer for %q and unit %q%w", uri.ID, unitName, secreterrors.SecretConsumerNotFound)
```

Comment:
This needs to be changed. I realise that it is not part of the PR but it is wrong.

```
fmt.Errorf("%w for id %q and unit %q", secreterrors.SecretConsumerNotFound, uri.ID, unitName)
```